proxies/shared/partials/Partial.Flows.CreateMessageBatchEndpoint.xml

```diff
@@ -1,6 +1,10 @@
 <Flow name="CreateMessageBatchEndpoint">
     <Description>Handle create message batch</Description>
     <Request>
+        <Step>
```

Comment:
I think this needs to be moved into a more generic flow rather than being in this specific flow.